freeipa/build.gradle

```diff
@@ -51,6 +51,9 @@ dependencies {
   implementation     group: 'io.opentracing.contrib',    name: 'opentracing-jaxrs2',                       version: opentracingJaxrs2Version
   implementation     group: 'io.opentracing.contrib',    name: 'opentracing-jdbc',                         version: opentracingJdbcVersion
 
+  implementation     group: 'org.apache.directory.api',  name: 'api-asn1-api',                         version: '2.0.0'
+  implementation     group: 'org.apache.directory.api',  name: 'api-asn1-ber',                         version: '2.0.0'
+  
```

Comment:
So what is this needed for? You copied the code, no?

Reply:
This is no longer needed with my upcoming change.